src/main/java/com/icthh/xm/ms/balance/service/BalanceService.java

```diff
@@ -93,4 +92,41 @@ public BalanceDTO findOne(Long id) {
     public void delete(Long id) {
         balanceRepository.delete(id);
     }
+
+    @Transactional
+    public void reload(ReloadBalanceRequest reloadRequest) {
+        log.info("Start reload balance with request {}", reloadRequest);
```

Comment:
@sergeysenja1992 do we need this log string here? seems it will be the almost same as logged by AOP